src/common/common.py

```
@@ -448,6 +634,10 @@ def display_large_dataframe(
     Returns:
         Index of selected row.
     """
+    if not PANDAS_AVAILABLE:
```

Ruff (0.8.2)

Undefined name `PANDAS_AVAILABLE`

(F821)